test/e2e/autoscaling/autoscaling_timer.go

```diff
@@ -80,12 +80,12 @@ var _ = SIGDescribe("[Feature:ClusterSizeAutoscalingScaleUp] [Slow] Autoscaling"
 				// Calculate the CPU request of the service.
 				// This test expects that 8 pods will not fit in 'nodesNum' nodes, but will fit in >='nodesNum'+1 nodes.
 				// Make it so that 'nodesNum' pods fit perfectly per node (in practice other things take space, so less than that will fit).
-				nodeCpus := nodes.Items[0].Status.Capacity[v1.ResourceCPU]
+				nodeCpus := nodes.Items[0].Status.Allocatable[v1.ResourceCPU]
```

Comment:
I suppose it depends of what we mean by "fit perfectly" - I'd say that allocatable is closer to it than capacity. I can of course rephrase it, what do you think would be more accurate?

Reply:
The text in parentheses says "(in practice other things take space, so less than that will fit)", so this is what I meant,  and this is the part that is no longer true. Maybe just removing that text parens is enough

Reply:
Right, I agree this may no longer always be the case. As this was already approved and has been merged since, is it OK if the fix goes in the next PR related to e2e tests?

Reply:
I think it is OK as long as it's a separate commit (or you can just create a dedicated PR)